core/src/main/java/org/vertexium/GraphBase.java

```diff
@@ -114,6 +114,28 @@ protected boolean isIncluded(Vertex v) {
     }
 
     @Override
+    public Iterable<Vertex> getVerticesInRange(Range idRange, Authorizations authorizations) {
+        return getVerticesInRange(idRange, FetchHint.ALL, authorizations);
+    }
+
+    @Override
+    public Iterable<Vertex> getVerticesInRange(Range idRange, EnumSet<FetchHint> fetchHints, Authorizations authorizations) {
+        return getVerticesInRange(idRange, fetchHints, null, authorizations);
+    }
+
+    @Override
+    public Iterable<Vertex> getVerticesInRange(final Range idRange, EnumSet<FetchHint> fetchHints, Long endTime, Authorizations authorizations) {
+        LOGGER.warn("Performing scan of all vertices! Override getVerticesInRange.");
```

Comment:
I find these warning messages really annoying when running the unit tests. Can we just log each message once?